src/cache/disk.rs

```diff
@@ -148,12 +148,14 @@ impl DiskCache {
     /// If it's not found or the entry is too old, the bool param will be false
     pub fn get_package_db_entry(&self, repo_url: &str) -> (PathBuf, bool) {
         let path = self.get_package_db_path(repo_url);
+
         if path.exists() {
-            let created = path
-                .metadata()
-                .expect("to work")
-                .created()
-                .expect("to have a creation time");
+            let metadata = path.metadata().expect("to work");
```

Comment:
I'm using a create called filetime (https://crates.io/crates/filetime) in eg src/fs.rs that i believe abstract over this kind of stuff, we should use it

Reply:
ok if you want to swap that over that'd be much appreciated thanks!